.github/workflows/build_packages.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
         if: endsWith(github.repository, 'emqx')
         run: |
           make -C source deps-all
+          rm source/rebar.lock
```

Comment:
This assumes all emqx deps have versions locked in rebar so that all artifacts are based on the same source, but I am unsure if it is true.

Reply:
@zmstone I remember you mentioned we don't rely on rebar.lock any more, is it still true?

Reply:
it is true that we do not need rebar.lock because we alway pin tags.

the reason we have to delete rebar lock is because git shallow clone will not work otherwise.